internal/exec/util/util.go

```diff
@@ -131,3 +131,13 @@ func PathExists(path string) (bool, error) {
 	}
 	return true, nil
 }
+
+// Helper function to concatenate the key files.
+func ZKeyHelper(l string) map[string]string {
```

Comment:
This really needs a more specific name. E.g. `ZkeySecureKeyRepoFiles`?

Reply:
Done